tests/control/ros/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+if(${FOUND_${PROJECT_NAME}_control_ros})
```

Comment:
`aikido_control_ros` is a local target so `FOUND_aikido_control_ros` wouldn't be defined. Instead, please use `if(TARGET "${PROJECT_NAME}_control_ros")`.

> **if(TARGET target-name)**
True if the given name is an existing logical target name such as those created by the add_executable(), add_library(), or add_custom_target() commands.